HPXMLtoOpenStudio/resources/hpxml_schema/HPXMLBaseElements.xsd

```diff
@@ -3748,14 +3775,6 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
-			<xs:element name="PackagedTerminalAirConditionerHeating">
-				<xs:complexType>
-					<xs:sequence>
-						<xs:element minOccurs="0" ref="extension"/>
-					</xs:sequence>
-					<xs:attribute name="dataSource" type="DataSource"/>
-				</xs:complexType>
-			</xs:element>
```

Comment:
You did everything correctly, but the CI is complaining because of this. Basically Yueyue proposed a change here, but her corresponding change to OS-HPXML is still in [a PR that hasn't been merged](https://github.com/NREL/OpenStudio-HPXML/pull/983). So if you revert this one change, it should make the CI happy.